pontoon/base/models/entity.py

```diff
@@ -892,6 +892,10 @@ def for_project_locale(
                         ~Q(resource__format="ftl")
                         & Q(**{f"string__{i}regex": rf"{y}{s}{y}"})
                     )
+                    | Q(
+                        ~Q(resource__format="ftl")
+                        & Q(**{f"string_plural__{i}regex": rf"{r}{y}{s}{y}{o}"})
```

Comment:
That doesn't seem right?
* This is not Fluent, so why do we need the prefix/postfix for ignoring context identifiers within the string?
* Why do we need a separate block instead of just adding line 897 after line 893?

Reply:
Apologies about the first bullet point!

As for the second one, the test case seemed to fail when I tried to coalesce both queries into one block, but it turned out I was incorrectly ANDing both queries instead of ORRing them.